frontend/src/components/form/base/ETextField.vue

```diff
@@ -5,8 +5,9 @@
     tag="div"
     :name="name"
     :vid="veeId"
-    :rules="veeRules"
+    :rules="dynamicRules"
     :required="required"
+    :mode="eagerIfChanged"
```

Comment:
I don't think this should be this eager:
https://pr3912.ecamp3.ch/login

When I start typing it immediately tells me that the e-mail is incorrect. I think this should be favorably on blur.

Reply:
True, I forgot to add a debounce. 
A debounce is a possible solution to that problem, but I am not that confident about the values I used since they affect a lot of input fields so I used the `apiPropsMixin.props.autoSaveDelay.default`. 

_There is a small issue where the user has a timeslot (800 ms) where the validation state isn't up to date and could (in theory) press a submit/sign-in button with invalid data in the input fields. I had a case at work where this caused some issues due to a lack of backend validation (which is not the case here, I tested it)._

Let me know what you think :)